samples/nrf9160/pdn_client/src/pdn_client.c

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <string.h>
+#include <pdn.h>
+
+void main(void)
+{
+	printk("The AT host sample started\n");
+
+	char address[] = "default";
+
+	void *pdn_handle = pdn_open();
+	if (!pdn_handle) {
```

Comment:
`pdn_handle == NULL` is more natural for pointers.

Reply:
will be aligned.